### PR TITLE
Suggestion for mfg python code

### PR DESCRIPTION
**Pull request by TheoCabannes**
- fix pylint errors: super with arguments.
- fix pylint errors: arguments-differ.
- fix pylint errors: invalid-name.
- fix pylint errors: method could be a function.
- fix pylint errors: no else return
- fix pylint errors: useless-object-inheritance
- do not have a default game info in crowd modelling python as Raphael suggested.
- use property x and t in crowd modelling python

### CODE REVIEW

File: open_spiel/python/mfg/algorithms/distribution.py [hunk 1]
```diff
@@ -19,6 +19,22 @@
...
 
+def type_from_states(states):
+  """Get node type of a list of states and assert they are the same."""
+  types = [state.get_type() for state in states]
+  for state_type in types:
+    assert types[0] == state_type, f'types: {types}'
+  return types[0]
+
+
+def is_terminal_from_states(states):
```
Comment (RaphaelMarinier) [inline open_spiel/python/mfg/algorithms/distribution.py:30]: We could actually either:
- get rid of this, and instead use type_from_states(states) == pyspiel.StateType.TERMINAL on line 71, or
- replace the body of this function with "return type_from_states(states) == pyspiel.StateType.TERMINAL"

File: open_spiel/python/mfg/games/crowd_modelling.py [hunk 2]
```diff
@@ -143,18 +144,16 @@ def _legal_actions(self, player):
     """Returns a list of legal actions for player and MFG nodes."""
     if player == pyspiel.PlayerId.MEAN_FIELD:
       return []
-    elif player == 0 and player == self.current_player():
+    if player == 0 and player == self.current_player():
```
Comment (RaphaelMarinier) [inline open_spiel/python/mfg/games/crowd_modelling.py:147]: DEFAULT_PLAYER_ID?

**Review by RaphaelMarinier — CHANGES_REQUESTED** [review]
Looks good overall.

File: open_spiel/python/mfg/games/crowd_modelling.py (hunk 2 above)
Comment (TheoCabannes) [reply open_spiel/python/mfg/games/crowd_modelling.py]: Done.

File: open_spiel/python/mfg/algorithms/distribution.py (hunk 1 above)
Comment (TheoCabannes) [reply open_spiel/python/mfg/algorithms/distribution.py]: I chose open 2.

**Review by RaphaelMarinier — APPROVED** [review]
LG

**Pull request merged**